<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2D2D30;
    color: #CCCCCC;
}

QLabel, QCheckBox {
    color: #D0D0D0;
}

QPushButton {
    background-color: #0078D7; /* Adjust the color code to match the blue you desire */
    color: white;
    border: none;
    border-radius: 2px;
    padding: 5px;
    margin: 2px;
    text-align: center;
}

QPushButton:hover {
    background-color: #005FA3;
}

QPushButton:pressed {
    background-color: #003A70;
}

QPushButton:disabled {
    background-color: #004C99;
    color: #7F7F7F;
}

QLineEdit {
    background-color: #2D2D30;
    border: 1px solid #555;
    border-radius: 2px;
    color: #D0D0D0;
    padding: 1px;
}

QRadioButton::indicator {
    border: 1px solid #555;
    width: 14px;
    height: 14px;
    border-radius: 7px;
    background-color: #2D2D30;
}

QRadioButton::indicator:checked {
    background-color: #3A3A3C;
    border: 1px solid #3A3A3C;
}

QRadioButton {
    spacing: 5px;
}

QSlider::groove:horizontal {
    border: 1px solid #444;
    height: 8px;
    background: #2D2D30;
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: #3A3A3C;
    border: 1px solid #5C5C5C;
    width: 18px;
    margin: -2px 0;
    border-radius: 9px;
}

QSlider::add-page:horizontal {
    background: #555;
}

QSlider::sub-page:horizontal {
    background: #3A3A3C;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="GB_mainwindow">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_app_title">
         <property name="text">
          <string>Blood sugar prediction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_graph_and_user">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_current_user">
            <property name="text">
             <string>Current user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="GB_user">
            <property name="title">
             <string>User</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QPushButton" name="pb_new_user">
               <property name="text">
                <string>New User</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pb_daily_values">
               <property name="text">
                <string>Input Daily Values</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GB_graphviewer">
         <property name="title">
          <string>Graphviewer</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGraphicsView" name="graphicsView_blood_insulin_graph"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
